src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -135,5 +135,26 @@ public static ScheduleTypes fromName(String name) {
         public static final String RECURRING_DV_EXPLANATION_TO_DV_NOTE_STARTER = "DV created by recurring DV: ";
         public static final String RECURRING_DETAILS_TAB_NAME = "RecurringDetails";
     }
+    
+    public static class AmazonWebServiceBillingConstants {
+        public static final String AWS_PROCESSING_DATE_PROPERTY_NAME = "AWS_PROCESSING_DATE";
+        public static final String AWS_FROM_ACCOUNT_PROPERTY_NAME = "AWS_FROM_ACCOUNT_NUMBER";
+        public static final String AWS_OBJECT_CODE_PROPERTY_NAME = "AWS_OBJECT_CODE";
+        public static final String AWS_DEFAULT_DOCUMENT_DESCRIPTION_PROPERTY_NAME = "AWS_DEFAULT_DOCUMENT_DESCRIPTION";
+        public static final String AWS_NOTE_HELP_TEXT_PROERTY_NAME = "AWS_NOTE_HELP_TEXT";
+        public static final String AWS_COMPENT_NAME = "LoadAWSBillsStep";
+        
+        public static final int PROCESSING_DAY_OF_MONTH = 15;
```

Comment:
This value is used to create a calendar object, and the day of month actually isn't used.  Since this will be run starting on the 15th of a month, 15 seemed like a reasonable number to go with.